src/hooks/api.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from 'react';
+
+const baseURL = process.env.REACT_APP_ApiPath;
+export default function ApiNinja() {
+  const [data, setData] = useState(null);
+
+  useEffect(() => {
+    fetch(`${baseURL}/?category=attitude`, {
+      method: 'GET',
+      headers: { 'X-API-KEY': '/tZkLGCPasBYhhpCUjjWCw==8PHIdKMaefWjcRht' },
+      contentType: 'application/json',
+    })
+      .then((res) => res.json())
+      .then((res) => setData(res));
+  }, []);
+
+  return ({
+    data,
```

Comment:
- Based on the requirements,
 
 ```
Show a loading state
Show an error state 
```

- [ ] Currently, when fetching data you don't check if the data is still loading or if there is an error, It would be better to use async and await to first check if your state is loading and when it has fetched you display data, also if fetch is not successful you should also display an error  message👍🏿 
-  You are currently showing a loading text in your quotes component but it would be better to do handle if data is loading where you are fetching 👍🏿